bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -163,6 +172,10 @@ if args.randomize_template_order:
 else:
     template_ids = range(tmin, tmax)
 
+# Only analyze templates which might have triggers based on
+# our 'tids_with_trigs'
+template_ids = numpy.intersect1d(tids_with_trigs, template_ids)
```

Comment:
Does this bit add significant effort in the case where we _don't_ want to only analyse certain templates, i.e. if `len(reader.file[ifo]['template_id']) > 2**27`

Can we add a flag in that case and therefore skip this intersect operation?

Reply:
Ok, done.